### ISSUE TITLE
Bootstrap without Chef-Client Run

### ISSUE THREAD

**Issue by sawanoboly** [higanworks/knife-zero#25]
UseCase: http://qiita.com/sawanoboly/items/4f363909615d8a76e9e5#comment-dc7a0303492056b66ca8

**Comment by sawanoboly** [comment]
-W (wry-run)つければ一応同じことできるな。

**Comment by patcon** [comment]
Could override this in our bootstrap context if we felt like it:
https://github.com/chef/chef/blob/d8172e646d9fbf43e57bca5e20d0ac352ba9a66a/lib/chef/knife/core/bootstrap_context.rb#L126-L133

**Comment by sawanoboly** [comment]
Thanks, 
This feature is assuming the update client.rb only.

**Comment by patcon** [comment]
:+1:

**Comment by patcon** [comment]
Oh hey, I think the more common flag name would be `skip_chef_run`, but in chef world, it seems the "no-" form is preferred.

What about a `--[no-]converge` flag for only the `knife zero bootstrap` command with an underlying var of `:bootstrap_converge`, that defaults to true? This would follow how `:use_sudo` works.

```rb
# .chef/knife.rb
knife[:bootstrap_converge] = false
```

```
# Does bootstrap without chef run
knife zero bootstrap foo.example.com

# Does bootstrap with chef run turned back on
knife zero bootstrap bar.example.com --converge
```

**Comment by sawanoboly** [comment]
OK, `--[no-]converge` and `bootstrap_converge` is not bad.

**Comment by sawanoboly** [comment]
I changed to `knife[:bootstrap_converge]` and `--[no-]converge`. And released as v1.7.1

**Comment by patcon** [comment]
You're awesome, @sawanoboly!
